pages/linux/toolbox-init-container.md

```diff
@@ -0,0 +1,24 @@
+# toolbox init-container
+
+> Initialize a running container.
+> More information: <https://www.mankier.com/1/toolbox-init-container>.
+
+- Pass gid as the user's numerical group ID from the host to the `toolbox` container:
```

Comment:
This command initializes the container with this specific GID, right? Then it should be more like:
```suggestion
- Initialize a running container and pass the user's GID from the host to the `toolbox` container:
```
But is this command even possible, since it cannot be executed on the host? And which `toolbox` container is it passed to?

Reply:
I haven't used a init-container before so I don't know the feasibility of the command.